rocAL/rocAL/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ find_package(RapidJSON QUIET)
 
 # Check for filesystem support
 include(CheckCXXSourceCompiles)
-set(CMAKE_REQUIRED_FLAGS "${CMAKE_CXX_FLAGS}")
+set(CMAKE_REQUIRED_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++17")
```

Comment:
https://github.com/ROCm/MIVisionX/blob/ae9c9ece492859bd50bf66c88e6cf5697630fd79/rocAL/rocAL/CMakeLists.txt#L271

adds the CXX Flags

Reply:
[https://github.com/ROCm/MIVisionX/blob/master/rocAL/rocAL/CMakeLists.txt#L47](url)
This line runs the std filesystem checks before the CXX flags are set so unless they are updated before this check, std filesystem check always fails